src/main/kotlin/no/nav/tilleggsstonader/soknad/prosessering/SendNotifikasjonTask.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.tilleggsstonader.soknad.prosessering
+
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.tilleggsstonader.soknad.soknad.SøknadService
+import no.nav.tilleggsstonader.soknad.soknad.domene.Søknad
+import no.nav.tilleggsstonader.soknad.varsel.DittNavKafkaProducer
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+@TaskStepBeskrivelse(taskStepType = SendNotifikasjonTask.TYPE, beskrivelse = "Send notifikasjon mottatt søknad")
+class SendNotifikasjonTask(
+    private val notifikasjonsService: DittNavKafkaProducer,
+    private val søknadService: SøknadService,
+) : AsyncTaskStep {
+
+    override fun doTask(task: Task) {
+        val søknad = søknadService.hentSøknad(UUID.fromString(task.payload))
+        val message = lagNotifikasjonsMelding()
+        val eventId = task.metadata["eventId"].toString()
+
+        notifikasjonsService.sendToKafka(søknad.personIdent, message, eventId)
+    }
+
+    private fun lagNotifikasjonsMelding(): String {
+        return "Vi har mottatt søknaden din om pass av barn."
+    }
```

Comment:
Hvis man legger in en when her på stønadstype så feiler denne vid kompilering når vi legger til en ny stønadstype. Kan fort bli noe som ikke blir testet når vi legger til ny stønadstype
```suggestion
    private fun lagNotifikasjonsMelding(søknad: Søknad): String {
        return when(søknad.type) {
            Stønadstype.BARNETILSYN -> "Vi har mottatt søknaden din om pass av barn."
        }
    }
```

Reply:
endring tatt inn